virttest/virsh.py

```diff
@@ -621,6 +624,93 @@ def kosher_args(remote_ip, uri):
         return True not in all_false
 
 
+class EventNotFoundError(Exception):
+    """
+    Error when certain event cannot be found.
+    """
+
+    def __init__(self, details=''):
+        self.details = details
+        Exception.__init__(self)
+
+    def __str__(self):
+        return str(self.details)
+
+
+class EventTracker(object):
+
+    @staticmethod
+    def start_get_event(vm_name):
```

Comment:
Is this method dedicated to  wait_event for use or exposed as public API?
something like we can use :
EventTracker.start_get_event(vm)
....
EventTracker.start_finish_event(vm)

Reply:
@chunfuwen, I suggest you think more. From my POV, aexpect.ShellSession is a thirdparty package, we'd better not add test specific functions into it.

Reply:
> @chunfuwen, From my POV, aexpect.ShellSession is a thirdparty package, we'd better not add test specific functions into it.
aexpect is maintained by lukas and clebergnu from Red Hat, part of avocado projects
https://github.com/avocado-framework/aexpect

Reply:
@chunfuwen I agree with @dzhengfy .
But, @chunfuwen feel free to add these 2 functions to aexpect if you'd like. But that should not be reason to block this pr.
So, I'd be glad to modify these functions after your commit to aexpect got merged. Thanks.